### ISSUE TITLE
The return of PhyloStyloTastic

### ISSUE THREAD

**Issue by gaurav** [OpenTreeOfLife/hackathon#17]
One of the projects I thought was really interesting back in Phylotastic II was PhyloStyloTastic (PST: http://www.evoio.org/wiki/PhyloStyloTastic). The idea behind PST is simple:
1. NeXML file containing metadata goes in.
2. The PST parser uses style information contained in a separate TSS file (which uses the same syntax as CSS) to figure out how to render the tree. The specification for TSS can be found at https://docs.google.com/document/d/13u-l2jn_xjARaNlNNCwa-v2Qu-hamIeOgYxV96pbeac/edit
3. It then passes these instructions on to a tree rendering program, allowing high-quality phylogenetic trees to be consistently rendered.

Apart from making it easy for particularly expressive tree designs to be reused, this would encourage richer metadata to be stored in NeXML files. IIRC, the PST team already implemented a basic TSS parser into Mesquite. Would anybody be interested in trying to implement one in another open-source tree builder (such as Newicktree: http://cbr.jic.ac.uk/dicks/software/newicktree/, Archaeopteryx: https://sites.google.com/site/cmzmasek/home/software/archaeopteryx, or the ETE toolkit: http://etetoolkit.org/), and see if we can make it shinier, for example by automatically adding [PhyloPic](http://phylopic.org/api/) images to OTUs.

**Comment by kcranston** [comment]
Hey Guarav! We've been talking about this idea in the context of the [Tree Illustrator ](https://github.com/OpenTreeOfLife/tree-illustrator) project. Definitely interested!

**Comment by mirsinarm** [comment]
It might also be useful to have a way of displaying different levels/types of support—maybe this already exists or is planned?  I've been looking through the documentation but haven't found anything yet.  I'm thinking that knowing whether a node is supported by multiple trees, by multiple genes, by certain bootstrapping levels, etc. could be useful for exploring the data or for generating publishable figures.

**Comment by chinchliff** [comment]
The updated API makes a node_info call available that will provide
information about supporting trees, etc. We are still working on the
documentation for this call, but you will be able to find it here later
today or tomorrow:

https://github.com/OpenTreeOfLife/opentree/wiki/Open-Tree-of-Life-APIs

**Comment by hlapp** [comment]
@rossmounce should be interested in this.

**Comment by rossmounce** [comment]
Yep. Will follow progress. On holiday in China atm. Am pleased github isn't blocked!

**Comment by daisieh** [comment]
I would also like to revisit this concept with some people who know more about standards and schemas; I think that the naive implementation of TSS could use a bit of extra love to standardize it.

In addition, I don't think we ever addressed the idea of branches moving or shifting and what happens to annotations when that happens.

**Comment by msrosenberg** [comment]
So as one of the original hackers of the TSS idea, I'd like to see it go forward, but the biggest problem in some ways is the completely lack of implementation. It's not that the TSS "standard" (as it were) can't be improved upon, but if no one is attempting to use it anywhere, it doesn't do us much good.
